packages/@tinacms/fields/src/Wysiwyg/useMarkdownTranslator.ts

```diff
@@ -17,14 +17,17 @@ limitations under the License.
 */
 
 import * as React from 'react'
-import { MarkdownTranslator } from './Translator'
+import { MarkdownTranslator, Format, DOMTranslator } from './Translator'
 import { Schema } from 'prosemirror-model'
 
-export function useMarkdownTranslator(schema: Schema) {
-  // TODO: Use `wysiwyg:markdown` plugins
-  const translator = React.useMemo(() => MarkdownTranslator.fromSchema(schema), [
-    schema,
-  ])
```

Comment:
It will be useful to rename `useMarkdownTranslator`.

Reply:
Oh, yes I meant to do that. Thanks